components/builder-notify/src/server.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{bldr_events::connection::{create_consumer,
+                                      EventBusConsumer},
+            config::Config,
+            error::Error,
+            hab_core::{config::ConfigFile,
+                       ok_warn}};
+use std::{path::PathBuf,
+          thread::{self},
+          time::Duration};
+
+const CFG_DEFAULT_PATH: &str = "/hab/svc/builder-notify/config/config.toml";
+const DEFAULT_QUEUE_NAME: &str = "builder_events";
+
+pub struct AppState {
+    eventconsumer: Option<Box<dyn EventBusConsumer>>,
```

Comment:
Probably a tad more idiomatic to call this field `event_consumer` (i.e., snake case).

(It also lends support to the idea that maybe we don't need `Bus` in the trait name.)

Reply:
:snake: cased in https://github.com/habitat-sh/builder/pull/1549/commits/261fcc5a3138e104852fa8476e681b947820d042